ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -3,684 +3,6 @@
 
```

Comment:
Please revert changes to this file and add the metrics you need either here or to a separate class/module. Not all of these metrics are 'participant metrics', and such refactoring should be done in a separate PR. As a bonus, you won't have a PR with 150+ files changed :)

Reply:
I now have confliciting messages about how the metrics are handled.

@stefanobaghino-da told me to move the metrics for the json api out into a seperate class/file.

Because I wanted to reuse the MetricsReporting class I abstracted the metrics definition and moved the content in here into a seperate class/file.

I can revert all this but this will result in code duplication so far I see the situation right now.

PS: I'm not that fond of refactoring 150 files either but sometimes refactoring results in that and so PR's with 150+ files changed will exist.

Reply:
Refactoring is fine, mixing it up with actual changes is usually less good of an idea. You make your life harder (since it’s usually harder to get those changes merged) and that of reviewers. E.g., this PR adds metrics to the JSON API, if we want to shuffle something else around for that, make a PR which does nothing but that, get that merged and then follow up with the actual change.

For changes to `Metrics.scala`, I would suggest to keep that file either completely untouched and only add the JSON API metrics in a separate file or add the JSON API metrics in there addressing @stefanobaghino-da’s comment separately.